utilities/tags/addTagsUsingCSV.py

```diff
@@ -0,0 +1,44 @@
+import csv
+import requests
+
+SEMGREP_API_KEY = ""
```

Comment:
Can you set this up to get the value from the env var or settings YAML? There are related routines in quite a few of the existing utilities for this which you can base it on. Same for all these files.